src/main/java/org/sagebionetworks/web/client/widget/profile/UserProfileEditorWidgetImpl.java

```diff
@@ -15,7 +14,7 @@
 public class UserProfileEditorWidgetImpl implements UserProfileEditorWidget, UserProfileEditorWidgetView.Presenter {
 	
 	public static final String PLEASE_ENTER_A_VALID_URL = "Please enter a valid URL";
-	public static final String FILE_WAS_SELECTED_BUT_NOT_UPLOADED = "File was selected but not uploaded.";
+	public static final String PLEAE_SELECT_AN_IMAGE = "File was determined not to be an image.";

```

Comment:
Change message to tell user what image file types were are looking for (png, jpg, ...)